roles/ceph-rgw/tasks/main.yml

```diff
@@ -48,3 +49,9 @@
     - containerized_deployment
   # Hard code this so we will skip the entire file instead of individual tasks (Default isn't Consistent)
   static: False
+
+- name: include rgw_pool_pgs.yml
+  include: rgw_pool_pgs.yml
+  when:
+    - create_pools is defined
```

Comment:
`create_pools` is not really meaningful, please use something like `rgw_create_pools`. Thanks

Reply:
Don't agree, this is the name of variable in the rgws.yml.sample , and not sure in how many places is called or even worst, if it is mentioned somewhere in the downstream documentation: 

https://github.com/ceph/ceph-ansible/blob/master/group_vars/rgws.yml.sample#L45

If you think this should be renamed, I think it should be done in a different PR.

Reply:
I guess it's a misunderstanding, this variable wasn't introduced in this patch so @jtudelag is just reusing it.
In any case, I agree with @leseb it should be renamed to avoid confusion.

I don't mind to either add a commit in this PR or in a dedicated one @jtudelag

Reply:
Fair enough, but adding a commit to change the name won't hurt, there is no need for a separate PR.
You have a change to do anyway, so please include this as well.
Thanks.

Reply:
@jtudelag any news?

Reply:
@jtudelag we have a [BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1550281) targeted for v3.0z4 linked to this patch, that would be awesome to get this patch merged very soon :-)

Reply:
@guits done ;)